web/source/osk/visualKeyboard.ts

```diff
@@ -400,10 +400,8 @@ namespace com.keyman.osk {
 
         // Note:  when embedded on Android (as of 14.0, at least), we don't get access to this.
         // Just the base key.
-        if (this.keyPending && this.keyPending.key) {
-          popupKeyMass = 3.0;
-          popupKeyID = this.keyPending.key.spec.coreID;
-        }
+        popupKeyMass = 3.0;
+        popupKeyID = subkey.coreID;
```

Comment:
Does this have to check that `subkey` has a value?

Reply:
Just checked to see how it actually functions; `subkey` actually _always_ has a value - and isn't always a subkey when the function starts.  It's simply the key-spec for whatever key the user selected.  That said, if we're finalizing a subkey gesture... it's a subkey.  So... the nomenclature could probably use a little work.

We could add a null guard on the block's condition though, just in case.

Reply:
> We could add a null guard on the block's condition though, just in case.

Good plan.